### PR TITLE
Final

### PR DESCRIPTION
**Pull request by AsmaAbusaa**

### CODE REVIEW

File: DoctorWho.Console/ProgramToExecPreviousExercise.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+void GetEnemiesNameByID(int id)
+{
+    var result = context.Episodes
+                  .Where(e => e.EpisodeId == id)
+                  .Select
+                  (a => new
+                  {
+                      EnemiesName = DoctorWhoCoreDbContext.GetEnemiesByEpisodeId(id)
+                  }
+                  ).ToList();
```
Comment (Hiba-Kh) [inline DoctorWho.Console/ProgramToExecPreviousExercise.cs:24]: is this the only way to do that? calling db function inside the context?

File: DoctorWho.Console/ProgramToExecPreviousExercise.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+void GetCompanionsNameByID(int id)
+{
+    var result = context.Episodes
+                .Where(e => e.EpisodeId == id)
+                .Select
+                (a => new
+                {
+                    CommpanionsName = DoctorWhoCoreDbContext.GetCompanionsByEpisodeId(id)
+                }
```
Comment (Hiba-Kh) [inline DoctorWho.Console/ProgramToExecPreviousExercise.cs]: same here

File: DoctorWho.Console/ProgramToExecPreviousExercise.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    conn.Open();
+    var reder = cmd.ExecuteReader();
+    while (reder.Read())
+    {
+        enemies.Add(new Enemy
+        {
+            EnemyId = reder.GetInt32("EnemyId"),
+            EnemyName = reder.GetString("EnemyName"),
+            Description = reder.GetString("Description"),
+        });
```
Comment (Hiba-Kh) [inline DoctorWho.Console/ProgramToExecPreviousExercise.cs:56]: nice try

File: DoctorWho.Console/ProgramToExecPreviousExercise.cs (hunk 1 above)
Comment (Hiba-Kh) [reply DoctorWho.Console/ProgramToExecPreviousExercise.cs:24]: I don't think it's a good way

Comment (AsmaAbusaa) [reply DoctorWho.Console/ProgramToExecPreviousExercise.cs:24]: I will change it to non-static function.

**Pull request merged**